app/src/main/java/org/molgenis/vibe/formats/Gene.java

```diff
@@ -19,7 +21,7 @@
     /**
      * The HGNC (HUGO Gene Nomenclature Committee) name.
      */
-    private String symbol;
+//    private String symbol; // Currently unused.
```

Comment:
Then why not remove it?

Reply:
I suggest leaving it in for now and doing a general code cleanup when:
* A more definitive output writer is implemented.
* This project is added to sonarcloud.